src/ruby/qps/proxy-worker.rb

```diff
@@ -155,6 +157,10 @@ def proxymain
     opts.on("-c", "--[no-]use_protobuf_c_extension", "Use protobuf C-extention") do |c|
       options[:c_ext] = c
     end
+    opts.on("-b" "--php_client_bin [FILE]",
+        "PHP client to execute; path relative to this script") do |c|
```

Comment:
nit: indentation of this line looks two spaces too far

Reply:
@apolcyn I'm not terribly proficient with Ruby. This line is a continuation of the line above though, so shouldn't it be doubly indented?

Reply:
In most other source files that are style checked, I believe such a continuation has two spaces, but I believe the style checker isn't running on this so I'm not actually certain

Reply:
I see. Done.